android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -745,11 +745,13 @@ private void initializePlayerSource(ReactExoplayerView self, DrmSessionManager d
         boolean haveResumePosition = resumeWindow != C.INDEX_UNSET;
         if (haveResumePosition) {
             player.seekTo(resumeWindow, resumePosition);
-        }
-        if (startPositionMs >= 0) {
+            player.setMediaSource(mediaSource, false);
+        } else if (startPositionMs == 0) {
+            player.setMediaSource(mediaSource, true);
+        } else if (startPositionMs > 0) {
             player.setMediaSource(mediaSource, startPositionMs);
         } else {
-            player.setMediaSource(mediaSource, !haveResumePosition);
+            player.setMediaSource(mediaSource, false);
```

Comment:
correct me if I am wrong, but the last else case should be:
player.setMediaSource(mediaSource, true );
?

```suggestion
            player.setMediaSource(mediaSource, true);
```

Reply:
This is the part that has changed.
If `resetPosition` is false, it use `getCurrentPosition()` internally.
I felt this behavior was more appropriate, so I fixed it.

Reply:
If you think it's better to guarantee the same behavior as before, I'll fix it.

Reply:
I was thinking the aim of this fix is to add the case with ` if (startPositionMs == 0) {` ?
because here it is the normal use case ?!

Reply:
I think I'm wrong and it would be better to just fix the bug case. Thanks :)